sources/iTermPreferencesBaseViewController.m

```diff
@@ -344,6 +376,19 @@ - (int)intForString:(NSString *)s inRange:(NSRange)range
     return val;
 }
 
+- (NSUInteger)uintForString:(NSString *)s inRange:(NSRange)range
+{
```

Comment:
Move the open brace to the end of the preceding line (this is the newer style, and I'm updating code as I go). Also, this method should be called `unsignedIntegerForString:inRange:`.